NoObjections/core/src/com/missionbit/game/sprites/Player.java

```diff
@@ -1,75 +0,0 @@
-//package com.missionbit.game.sprites;
```

Comment:
Talk to your group about whether you guys are going to keep a "Dude" class. If so, you need to refactor the Hero class to Dude.